units/patch_intent_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/evergreen-ci/evergreen/db"
 	"github.com/evergreen-ci/evergreen/mock"
 	"github.com/evergreen-ci/evergreen/model"
-	"github.com/evergreen-ci/evergreen/model/admin"
+	
```

Comment:
I'd avoid these blank lines in the import blocks

Reply:
fixed